### PR TITLE
Magdalena/forms editing

### PR DESCRIPTION
**Pull request by MiaArnix**

### CODE REVIEW

File: src/frontend/src/app/pages/profile-edit/admin-personal-info-edit/admin-personal-info-edit.component.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      </div>
+    </form>
+  `,
+})
+export class AdminPersonalInfoEditComponent implements OnChanges {
+  @Input({ required: true })
+  public userProfile!: AdminProfile;
+
+  @Output()
+  public formSave = new EventEmitter<void>();
```
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/admin-personal-info-edit/admin-personal-info-edit.component.ts]: Rename to `close`.

File: src/frontend/src/app/pages/profile-edit/admin-personal-info-edit/admin-personal-info-edit.component.ts
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/admin-personal-info-edit/admin-personal-info-edit.component.ts:1]: rename to `admin-personal-info-form`

File: src/frontend/src/app/pages/profile-edit/admin-personal-info-view/admin-personal-info-view.component.ts
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/admin-personal-info-view/admin-personal-info-view.component.ts:1]: rename to `admin-personal-info`

File: src/frontend/src/app/pages/profile-edit/admin-personal-info-view/admin-personal-info-view.component.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      </button>
+    </div>
+  `,
+})
+export class AdminPersonalInfoViewComponent {
+  @Input({ required: true })
+  public userProfile!: AdminProfile;
+
+  @Output()
+  public formEdit = new EventEmitter<void>();
```
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/admin-personal-info-view/admin-personal-info-view.component.ts]: Rename to `edit` or `editClick`

File: src/frontend/src/app/pages/profile-edit/admin-profile/admin-profile.component.ts [hunk 3]
```diff
@@ -60,92 +29,34 @@ export interface AdminProfileForm {
...
-      username: new FormControl('', {
-        nonNullable: true,
-        validators: [Validators.required, Validators.minLength(3)],
-      }),
-      bio: new FormControl('', {
-        nonNullable: true,
-        validators: [Validators.required],
-      }),
-    });
+  public editForm(): void {
```
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/admin-profile/admin-profile.component.ts]: Rename to `showForm`

File: src/frontend/src/app/pages/profile-edit/admin-profile/admin-profile.component.ts [hunk 4]
```diff
@@ -60,92 +29,34 @@ export interface AdminProfileForm {
...
-    const formValues = this.profileForm.value;
-
-    const profileUpdate: AdminProfileUpdate = {
-      role: UserRole.Admin,
-      username: formValues.username,
-      bio: formValues.bio,
-    };
-
-    this.profileService.saveProfile(profileUpdate);
+  public stopEditing(): void {
```
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/admin-profile/admin-profile.component.ts]: rename to `hideForm`

File: src/frontend/src/app/pages/profile-edit/reviewer-profile/reviewer-profile.component.spec.ts [hunk 5]
```diff
@@ -1,24 +1,16 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
 
-import { ProfileService, UserRole } from '~core/state';
-import {
-  ProfileServiceMock,
-  profileServiceMockFactory,
-} from '~core/state/profile/profile.service.mock';
+import { UserRole } from '~core/state';
+import {} from '~core/state/profile/profile.service.mock';
```
Comment (nathanosdev) [inline src/frontend/src/app/pages/profile-edit/reviewer-profile/reviewer-profile.component.spec.ts]: Delete

File: src/frontend/src/app/pages/profile-edit/admin-personal-info-edit/admin-personal-info-edit.component.ts (hunk 1 above)
Comment (MiaArnix) [reply src/frontend/src/app/pages/profile-edit/admin-personal-info-edit/admin-personal-info-edit.component.ts]: set to "formClose" due to ESLint check: "Output bindings, including aliases, should not be named as standard DOM events"

**Review by nathanosdev — APPROVED** [review]

**Pull request merged**